includes/polling/os/ciscowlc.inc.php

```diff
@@ -75,8 +64,122 @@
 }
 
 $fields = array(
-	'wificlients' => $numClients
+    'wificlients' => $numClients
 );
 
 rrdtool_update($wificlientsrrd, $fields);
 $graphs['wifi_clients'] = true;
+
+
+$ap_db = dbFetchRows('SELECT * FROM `access_points` WHERE `device_id` = ?', array($device['device_id']));
+
+foreach ($radios as $key => $value) {
+ 
+    $indexName = substr($key, 0, -2);
+
+    $channel       = str_replace('ch', '', $value['bsnAPIfPhyChannelNumber']);
+    $mac           = str_replace(' ', ':', $stats[$indexName]['bsnAPDot3MacAddress']);
+    $name          = $stats[$indexName]['bsnAPName'];
+    $numasoclients = $value['bsnApIfNoOfUsers'];
+    $radionum      = array_pop(explode('.',$key));
```

Comment:
Sorry I've only just realised I see the scrut comments from a plugin :/

`explode('.', $key) cannot be passed to array_pop() as the parameter $array expects a reference.`